app/javascript/react/screens/App/Overview/screens/MappingWizard/MappingWizardSelectors.js

```diff
@@ -1,10 +1,13 @@
-export const mappingWizardOverviewFilter = overview => ({
-  hideMappingWizard: overview.hideMappingWizard
-});
+import { selectKeys } from 'patternfly-react';
 
-export const mappingWizardFormFilter = form => ({
-  mappingWizardGeneralStep: form.mappingWizardGeneralStep,
-  mappingWizardClustersStep: form.mappingWizardClustersStep,
-  mappingWizardDatastoresStep: form.mappingWizardDatastoresStep,
-  mappingWizardNetworksStep: form.mappingWizardNetworksStep
-});
+export const mappingWizardOverviewFilter = overview =>
```

Comment:
i think it was mentioned in a recent Foreman PR (and in previous PF React Community meetings) that using a "memoized" selector is a performance boost. 
This library was mentioned (and currently evaluating that elsewhere):
https://github.com/reactjs/reselect

So for that reason... i don't think it makes sense to introduce `selectKeys`...but maybe a new memoized selector in pf-react later on... for now, can we just keep the current convention?

Reply:
Sure. Hadn't thought about the performance impact, I'll change this back.